runtime/gpufortrt/src/gpufortrt_api_core.f90

```diff
@@ -292,4 +292,31 @@ function gpufortrt_use_device_c_impl(hostptr,condition,if_present) &
     resultptr = gpufortrt_use_device_c_impl(hostptr,opt_if_arg,opt_if_present_arg)
   end function
 
+  function gpufortrt_is_present(data_arg, bytes) result(isOnDeviceMemory) 
+    use iso_c_binding
```

Comment:
If not intended (in this case it shouldn't), try not to forget the `implicit none` after the `use` statements and before the first declaration / line of code.
Otherwise, Fortran's default implicit naming rules are active, which based on the first letter of an identifier
will assign a default integer or real type.

Reply:
fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/59d4293c415a47f93867135dddb827fb1fd7a4ae